cmakelib/QuickCppLibUtils.cmake

```diff
@@ -55,6 +55,7 @@ endfunction()
 function(list_filter listname op regexqualifer regex)
   if(CMAKE_VERSION VERSION_GREATER 3.59)
     list(FILTER ${ARGV})
+    set(${listname} ${${listname}} PARENT_SCOPE)
```

Comment:
Good catch!
However, I suspect that a “.” is missing in the version check, i.e. it should probably have said:
```patch
- if(CMAKE_VERSION VERSION_GREATER 3.59)
+ if(CMAKE_VERSION VERSION_GREATER 3.5.9)
```
Given that we already require CMake version 3.9, we might as well completely nuke the `else()` part and add a deprecation comment to the function.

Reply:
As you said, now we required CMake version 3.9, how about just use `list(FILTER ${ARGV})` and set `PARENT_SCOPE` in this function since the else part will never be reached?

Reply:
@JackBoosY yes, that's what I tried to suggest 👍

Reply:
I'm okay with removing the older cmake polyfill entirely.

Reply:
Done